src/viam/app/data_client.py

```diff
@@ -60,6 +62,54 @@ class DataClient:
     `ViamClient`.
     """
 
+    class TabularData:
```

Comment:
Returning the actual proto type for tabular data requests seemed like a bad idea because it's awkward to navigate and has a struct (which we wanted to get rid of. But, `time_requested` and `time_received` are their own fields, and I thought returning a 4ple would be super ugly. So, I created a class to represent meaningful tabular data, and created an analogous class for binary data for consistency's sake.

Happy to revisit the type question if there's pushback against creating these shadow classes.